src/components/scenes/SendScene.tsx

```diff
@@ -381,7 +381,11 @@ class SendComponent extends React.PureComponent<Props, State> {
   }
 
   renderScamWarning() {
-    return <ScamWarningCard marginRem={[1.5, 1]} />
+    const { recipientAddress } = this.state
+    if (!recipientAddress) {
```

Comment:
Don't use vague boolean checks in IF statements. Do a check that returns a true boolean. ie `recipientAddress != null` or `recipientAddress != ''` or whatever is most appropriate. This should have caused a typescript error.

Also, function doesn't have a defined return value. Not sure how this file is passing typescript checks, but all functions should have return types.

Reply:
Fixed the ambiguous boolean issue & will note that for the future. For some reason, it doesn't trigger typescript errors in my vscode config, will look into that.

For the null return, I was following the pattern that similar `render*()` functions use within the SendScene, such as [renderError()](https://github.com/EdgeApp/edge-react-gui/blob/05a38e2c862f33a8b377bff5f9b229ab6bacebf7/src/components/scenes/SendScene.tsx#L426-L432),